nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/ChartProcessor.java

```diff
@@ -1328,6 +1328,10 @@ public void execScript(String script, boolean doProcess, boolean reloadData) {
         Processor.getProcessor().clearProcessorError();
         ProcessOps process = getProcess();
         process.clearOps();
+        if (processorController == null) {
+            System.out.println("null proc");
+            return;
+        }
```

Comment:
Moved this above the first access to processController, since if it was null it would cause an exception on the next line, just not sure if that is okay to return early here or throw an exception of some kind?